provisioner/terraform/tfparse/tfparse.go

```diff
@@ -290,6 +305,18 @@ func (p *Parser) CoderParameterDefaults(ctx context.Context, varsDefaults map[st
 			continue
 		}
 
+		var found bool
+		needle := strings.Join([]string{"data", dataResource.Type, dataResource.Name}, ".")
```

Comment:
As far as I can tell, there is no way to break a variable reference over multiple lines e.g.

```
data.\
coder_parameter.\
foo.\
value
```